posts/2024/2024_06_03_kaustav_week_1.rst

```diff
@@ -0,0 +1,49 @@
+My Journey Continues: Week 2 Progress with DIPY
+===============================================
```

Comment:
Week 2 ?   week 1 I believe. your filename is week 1, and the previous week was week 0

Reply:
You are right, Changed.